activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+*   Replace `db:schema:load_if_ruby` and `db:structure:load_if_sql` with meta task `db:load_schema_or_structure`
+
+    Fixes #29128
```

Comment:
Can you add a period to the end of this line?

I only mention this because I've seen it fixed up by maintainers before, as in https://github.com/rails/rails/commit/0017866341e6ee085987173b0b586e8b7e8fee65.